public/js/main.js

```diff
@@ -645,6 +645,37 @@ function alignRightHeaders() {
     });
 }
 
+function addSingleQuoteToCsvIfNeeded(str) {
+  const specialChars = ['=', '+', '-', '@', '\t', '\r'];
```

Comment:
@davidrapin note: I also checked with an empty space at the beginning of the value, it's not an issue in google sheets, however it is interpreted as a formula in Numbers on Max 😬. 
Should I include ' ' in those values? Or add something like:
```
if (specialChars.some(char => str.startsWith(' ' + char))) {
```
or is it out of scope?

Reply:
does the problem exists only with one space, or also with multiple?
If it only exists with one space your proposal makes sense, otherwise we need to introduce regular expressions.
depending on the complexity it’s tempting to keep the scope of the fix to the most obvious problem and ignore more “corner” ones.

Reply:
indeed, it's also for multiple spaces. Should we keep the scope to specs found online, ie what is already implemented?